test/functions_test.py

```diff
@@ -201,7 +201,7 @@
 >>> load_config("test123456789")
 [Error] Failed to load config file!
 >>> loaded_config = check_for_config(input_func_conf_test1)
-Config files detected in current directory are listed below:
+Config files detected in current directory are listed below :
```

Comment:
Also, fix them accordingly.

Reply:
fixed in https://github.com/sepandhaghighi/pyrgg/pull/170/commits/ce9d852ec1025fb4dfaf9fa784824e708712a389